spec/auth.md

```diff
@@ -0,0 +1,59 @@
+# Echo Server Authentication
+
+## Relay <--> Echo Server
+
+We use Ed25519 to verify that requests from the Relay network are valid and intended for the Echo Server Instance that has received them.
+
+### Implementation
+
+Echo Server generates<sup>[[1]](#generating-keys)</sup> an Ed25519 key-pair and stores that in-memory - or in files specified using environment variables. The generated
+public key is then sent to the Relay network along with a cloud app project id and the public url (both from environment variables) so that
```

Comment:
`is then send to the Relay Network`.

The Relay network doesn't exist yet. Can you clarify which are going to be consumers of this public key ?

Reply:
I'll reword to make it make sense. Its actually just the rust relay...

Reply:
From the proposal: it seems that it should be the all the relay servers that query the public key: https://docs.google.com/document/d/1DPmAea-kC7wWVgj5H6eKFVpYzRXCUk8FvhosEMt0aK0/edit#

Reply:
Yeah, thats correct I have changed the .md file to reflect that too